[UI] Better stats overview for nodes and stores

maxlang opened the pull request:
- Added new stats and a rollup to the nodes and stores pages
- moved graphs into the nodes and stores overview pages
- added cluster icon

Old page:
![screen shot 2015-12-10 at 4 50 16 pm](https://cloud.githubusercontent.com/assets/3691886/11729648/3dcd9a84-9f5e-11e5-92ab-9276c3e13671.png)

New page:
![screen shot 2015-12-10 at 4 47 03 pm](https://cloud.githubusercontent.com/assets/3691886/11729647/3dc9db24-9f5e-11e5-87a9-3a2066db9604.png)

<!-- Reviewable:start -->

[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/3401)

<!-- Reviewable:end -->

Commit:
better stats overview for nodes and stores

mrtracy commented:
Review status: 0 of 13 files reviewed at latest revision, 4 unresolved discussions.

---

*[ui/styl/partials/table.styl, line 109 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/3401#-K5XTTLRDTQX0g8Bt4tt:-K5XTTLRDTQX0g8Bt4tu:1820469047) ([raw file](https://github.com/cockroachdb/cockroach/blob/f5a53f3a824709a7f5e35544fbdd4d0b29c32e74/ui/styl/partials/table.styl#L109)):*
Keep in mind that many of these colors might have to be changed in international versions down the line; they might not be the same color.  The `$secondary-red` is totally fine, but it is not correct for the element being styled to have a subclass of `.red`.  Instead, subclasses like `.warn` and `.healthy` would be better.

---

*[ui/ts/components/table.ts, line 144 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/3401#-K5XdLEx4aIfJXg3WMCA:-K5XdLEx4aIfJXg3WMCB:-681588025) ([raw file](https://github.com/cockroachdb/cockroach/blob/f5a53f3a824709a7f5e35544fbdd4d0b29c32e74/ui/ts/components/table.ts#L144)):*
Good simple solution here.

---

*[ui/ts/models/status.ts, line 33 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/3401#-K5XcEtsmaEQv3UtV4Dp:-K5XcEtsmaEQv3UtV4Dq:34327555) ([raw file](https://github.com/cockroachdb/cockroach/blob/f5a53f3a824709a7f5e35544fbdd4d0b29c32e74/ui/ts/models/status.ts#L33)):*
Maybe a small comment about the output.

These also don't have to be if/else, they can just be subsequent if statements because they are short-circuited by a `return`.  This also means you can compute the _1min moment after the _10min case.

---

*[ui/ts/util/table.ts, line 15 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/3401#-K5X_gBF7EQWLieAzWG5:-K5X_gBF7EQWLieAzWG6:-806379972) ([raw file](https://github.com/cockroachdb/cockroach/blob/f5a53f3a824709a7f5e35544fbdd4d0b29c32e74/ui/ts/util/table.ts#L15)):*
I really don't like this import. Utils should not be importing store status.   

This is so specific to StoreStatus/NodeStatus that it's not really a util anymore.  The util here is the `_.reduce` function.  These should just go right next to the `staleStatus` function you created.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/3401)*
<!-- Sent from Reviewable.io -->

maxlang commented:
Review status: 0 of 12 files reviewed at latest revision, 4 unresolved discussions.

---

*[ui/styl/partials/table.styl, line 109 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/3401#-K5XTTLRDTQX0g8Bt4tt:-K5_3jeAfJVC_gS6cXHX:501014754) ([raw file](https://github.com/cockroachdb/cockroach/blob/f5a53f3a824709a7f5e35544fbdd4d0b29c32e74/ui/styl/partials/table.styl#L109)):*
Changed to healthy, stale, and missing for now. We can always re-word it once we have a better heuristic too.

---

*[ui/ts/models/status.ts, line 33 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/3401#-K5XcEtsmaEQv3UtV4Dp:-K5_AWdhhmstOMjNd3TR:-1182737452) ([raw file](https://github.com/cockroachdb/cockroach/blob/f5a53f3a824709a7f5e35544fbdd4d0b29c32e74/ui/ts/models/status.ts#L33)):*
Simplified and added a comment

---

*[ui/ts/util/table.ts, line 15 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/3401#-K5X_gBF7EQWLieAzWG5:-K5_AIAChOi729chAxSO:996093096) ([raw file](https://github.com/cockroachdb/cockroach/blob/f5a53f3a824709a7f5e35544fbdd4d0b29c32e74/ui/ts/util/table.ts#L15)):*
moved to status.ts

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/3401)*
<!-- Sent from Reviewable.io -->

mrtracy commented:
LGTM

---

Review status: 0 of 12 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/3401#-:-K5gusypxj1bMgTn5Dqz:676834233)*
<!-- Sent from Reviewable.io -->
